GameServerSource/Assets/Packages/ChatServer/Commands/ChatServerMessagingCommands.cs

```diff
@@ -50,16 +49,18 @@ public override void Execute()
                     {"GameId", ServerSettings.GameId},
                     {"CreatedBy", Message.MemberId }
                 }
-            });
-            WritePlayerEventSignal.Dispatch(new WriteServerPlayerEventRequest()
+            }, null, null);
```

Comment:
I think a lot of these "null, null" could be funneled into a relay function, which allows optional logging or subscription to these API calls succeeding or failing. Right now, a ton of these can be failing, and it would be invisible.